site/content/docs/5.2/forms/select.md

```diff
@@ -25,6 +25,12 @@ Custom `<select>` menus need only a custom class, `.form-select` to trigger the
 
 You may also choose from small and large custom selects to match our similarly sized text inputs.
 
+{{< ods-incompatibility-alert >}}
+These select sizes variants, with a **height different than 40px**, should not be used because they do not respect the Orange Design System specifications.
```

Comment:
Already in the rest of this design callout message. I'm removing the link towards ODS Pages section because it has changed and is not relevant anymore here.